app/src/main/java/com/eveningoutpost/dexdrip/localeTasker/receiver/FireReceiver.java

```diff
@@ -162,8 +162,14 @@ public void onReceive(final Context context, final Intent intent) {
                             break;
 
                         case "SNOOZE":
-                            AlertPlayer.defaultSnooze();
-                            JoH.static_toast_long("SNOOZE from Tasker");
+                            int minutes = -1; // default
+                            if (message_array.length > 1) {
```

Comment:
If this change is not related to the intend of the PR, please remove it.

Reply:
Done. Removed the tasker related changes -- rebased and removed the commit. I'll publish a separate CR with this change and some other tasker additions.